src/components/comment.jsx

```diff
@@ -1,13 +1,40 @@
+import { deleteComment } from "../utils/api-calls";
 import { formatDate } from "../utils/format-article-date";
 import Divider from "./divider";
+import "../styles/delete-comment.css"
 
-export default function Comment({ comment }) {
+export default function Comment({ comment, user, setDeleted, deleted }) {
+  const handleDelete = (comment_id) => {
+    deleteComment(comment_id).then(() => {
+      setDeleted(comment_id);
+    });
+  };
 
-    return (
+  return (
```

Comment:
If 2 comments are deleted only the latest deleted comment is removed from the page. A refresh shows both as deleted.